src/Database/Connection.php

```diff
@@ -756,6 +757,9 @@ public function inTransaction(): bool
     public function quote($value, $type = 'string'): string
     {
         [$value, $type] = $this->cast($value, $type);
+        if (!is_string($value)) {
+            throw new DatabaseException('Unable to quote non-string values.');
```

Comment:
This one might be unnecessary since it should throw a php error.

Reply:
We could skip it for now as PHP should be taking care of this.